### PR TITLE
Update code ownership

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Updating code ownership to add all .NET crew to as owners.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
-Microsoft.Azure.Cosmos/src/Json @bchong95 @j82w
-Microsoft.Azure.Cosmos/src/Query @bchong95 @j82w
-Microsoft.Azure.Cosmos/src/CosmosElements @bchong95 @j82w
-Microsoft.Azure.Cosmos/src/SqlObjects @bchong95 @j82w
-Microsoft.Azure.Cosmos/src/Linq @khdang @j82w
+Microsoft.Azure.Cosmos/src/Json @bchong95 @j82w @sboshra
+Microsoft.Azure.Cosmos/src/Query @bchong95 @j82w @sboshra
+Microsoft.Azure.Cosmos/src/CosmosElements @bchong95 @j82w @sboshra
+Microsoft.Azure.Cosmos/src/SqlObjects @bchong95 @j82w @sboshra
+Microsoft.Azure.Cosmos/src/Linq @khdang @j82w @sboshra @bchong95
```
Comment (bchong95) [inline CODEOWNERS:14]: Im not a Linq guy

File: CODEOWNERS [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
 Microsoft.Azure.Cosmos/src/Spatial @lalithamv @j82w
```
Comment (bchong95) [inline CODEOWNERS:15]: add sboshra here like the rest on top

File: CODEOWNERS (hunk 1 above)
Comment (j82w) [reply CODEOWNERS:14]: This allows you to sign off for any linq related PRs.

File: CODEOWNERS (hunk 2 above)
Comment (j82w) [reply CODEOWNERS:15]: sboshra isn't a required reviewer for the SDK.

Comment (bchong95) [reply CODEOWNERS:15]: I meant add sboshra for Spatial.

File: CODEOWNERS (hunk 1 above)
Comment (bchong95) [reply CODEOWNERS:14]: yeah that works for me

**Pull request merged**